packages/get-platform/package.json

```diff
@@ -24,7 +24,13 @@
   "dependencies": {
     "@prisma/debug": "workspace:*",
     "chalk": "4.1.2",
+    "execa": "5.1.1",
+    "escape-string-regexp": "4.0.0",
+    "fs-jetpack": "5.1.0",
+    "replace-string": "3.1.0",
+    "strip-ansi": "6.0.1",
     "terminal-link": "2.1.1",
+    "tempy": "1.0.1",
```

Comment:
If these new dependencies are only for tests, then we can move them to `devDependencies`

Reply:
Right call. Let me fix that.